### PR TITLE
Feature/fix testing

### PR DESCRIPTION
**Pull request by KarimAl-Rashdan**
## What does this merge do? Check all that apply.
- [ ] Adds a new feature
- [X ] Improves an existing feature
- [ ] Fixes a bug
- [X ] Cleans up (logs, tests, etc.)

## How has this been tested?
npm test
## Description:
Briefly, but clearly describe what you did, and explain any new code.
I updated the testing to reflect all current methods
## Issues:
Are there any concerns, issues, or bugs in this branch? If so describe them.
N/A
## Where to find Changes:
Files:__SleepRepository, ActivityRepository_____________
Methods:______________
line Numbers if possible: ________________________________________ 

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

**Review by BHaddad1 — APPROVED** [review]
Thanks for working on this!

File: src/scripts.js [hunk 1]
```diff
@@ -304,7 +304,7 @@ function updateSleepData() {
     
     sleepWeek.innerHTML += `<ul>
       <li>${dayActivity.date}: </li>
-      <li>Hours Slept: ${dayActivity.hoursSlept}</li>
```
Comment (AndrewCerveny) [inline src/scripts.js:307]: until we figure out what text/ font family remember this is in there and will eventually need to be readdressed.

**Review by AndrewCerveny — APPROVED** [review]
Look good, 
rogue console.logs()  HYDRO REPo(17)
ActivityRepo(73)
63 passing test! Well Done!

**Comment by AndrewCerveny** [comment]
I made comments to go back to and see if things needed to be fixed.

**Pull request merged**